tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -21,17 +21,14 @@ Microsoft.Data.SqlClient.SqlTransaction
 Microsoft.Data.SqlClient.SqlParameterCollection
 Microsoft.Data.SqlClient.SqlClientFactory
 
-When running on Windows, this library has a dependency on Microsoft.Data.SqlClient.SNI on .NET Framework and runtime.native.System.Data.SqlClient.sni on .NET Core, which requires the Microsoft Visual C++ Redistributable to be installed:
-https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads
-
 When using NuGet 3.x this package requires at least version 3.4.</description>
     <releaseNotes>https://go.microsoft.com/fwlink/?linkid=2090501</releaseNotes>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <tags>sqlclient microsoft.data.sqlclient</tags>
     <dependencies>
       <group targetFramework="net46">
         <dependency id="System.Data.Common" version="4.3.0" />
-        <dependency id="Microsoft.Data.SqlClient.sni" version="1.1.0" />
+        <dependency id="Microsoft.Data.SqlClient.sni" version="[1.1.0, 1.2.0)" />
```

Comment:
@David-Engel 

_Highlighting this additional change:_
I'll also port this to 'master' branch, once you confirm!

Reply:
Why we need the upper bound here?

Reply:
Thanks! I got it confused with auto-fetch of SNI package dependency, but no upper bound is needed.

Reply:
Confirmed range with upper bound, after internal discussion.